distrobuilder/main.go

```diff
@@ -651,6 +651,15 @@ ExecStart=-/usr/lib/systemd/systemd-sysctl
 EOF
 }
 
+# fix_systemd_users_credentials overrides the systemd-sysusers.service and disables credentials.
+fix_systemd_users_credentials() {
+	mkdir -p /run/systemd/system/systemd-sysusers.service.d
+	cat <<-EOF > /run/systemd/system/systemd-sysusers.service.d/zzz-lxc-override.conf
+[Service]
+LoadCredential=
```

Comment:
@monstermunchkin 
Any reason not to override it to empty for everything?

Reply:
Actually, no :)